src/test/regress/expected/coordinator_shouldhaveshards.out

```diff
@@ -624,8 +624,10 @@ select create_distributed_table('test_index_creation1', 'tenant_id');
 CREATE INDEX ix_test_index_creation5 ON test_index_creation1
 	USING btree(tenant_id, timeperiod)
 	INCLUDE (field1) WHERE (tenant_id = 100);
-NOTICE:  executing the command locally: CREATE  INDEX   ix_test_index_creation5_1503042 ON coordinator_shouldhaveshards.test_index_creation1_1503042 USING btree (tenant_id ,timeperiod ) INCLUDE (field1 )WHERE (tenant_id = 100)
-NOTICE:  executing the command locally: CREATE  INDEX   ix_test_index_creation5_1503045 ON coordinator_shouldhaveshards.test_index_creation1_1503045 USING btree (tenant_id ,timeperiod ) INCLUDE (field1 )WHERE (tenant_id = 100)
+NOTICE:  executing the command locally: CREATE  INDEX   ix_test_index_creation5_1503042 ON  coordinator_shouldhaveshards.test_index_creation1_1503042 USING btree (tenant_id ,timeperiod ) INCLUDE (field1 ) WHERE (tenant_id = 100)
```

Comment:
creating an index shouldn't trigger `worker_fix_partition_shard_index_names` for the other indexes.

Reply:
fixed